selenium-system-test/src/test/resources/features/accountHolder/Account holder changes their email.feature

```diff
@@ -1,6 +1,6 @@
 Feature: Account holder changes their email
 
-  @HTS-876
+  @HTS-876 @VerifyHandF
```

Comment:
@Jackie1988 Can we change the tag to be @VerifyHeaderAndFooter, please?

Reply:
ok done thanks